Package.swift

```diff
@@ -1,21 +1,16 @@
-// swift-tools-version:3.1
+// swift-tools-version:4.0
 
 import PackageDescription
-import Foundation
-
-// HACK from https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/Package.swift
-var isSwiftPMTest: Bool {
```

Comment:
I dont think this stuff is needed anymore...

Reply:
yup